src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/api/controller/SysconSyncController.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.digital.hmpps.personrecord.api.controller
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.tags.Tag
+import jakarta.validation.constraints.NotBlank
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus.CREATED
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.personrecord.api.constants.Roles
+import uk.gov.justice.digital.hmpps.personrecord.api.model.sysconsync.Prisoner
+import uk.gov.justice.hmpps.kotlin.common.ErrorResponse
+
+@Tag(name = "Syscon Sync")
+@RestController
+@PreAuthorize("hasRole('${Roles.QUEUE_ADMIN}')") // TODO change me :-)
+class SysconSyncController {
+
+  @Operation(description = "Create a prison record")
+  @PutMapping("/syscon-sync/{prisonNumber}")
```

Comment:
Also should have a post ?  and possibly a merge end-point ?

Reply:
yes definitely 
Delete too
Model for post will be the same - the aim of this is just to get the model in place so Syscon can start work

Reply:
Deletes are manual via a script I believe.  So not sure we will need to expose an API for that.